code/modules/mob/living/simple_animal/hostile/regalrat.dm

```diff
@@ -0,0 +1,151 @@
+/mob/living/simple_animal/hostile/regalrat
+	name = "regal rat"
+	desc = "An evolved rat, created through some strange science. It leads nearby rats with deadly efficiency to protect its kingdom. Not technically a king."
+	icon_state = "regalrat"
+	icon_living = "regalrat"
+	icon_dead = "regalrat_dead"
+	gender = NEUTER
+	speak_chance = 0
+	turns_per_move = 5
+	maxHealth = 70
+	health = 70
+	see_in_dark = 5
+	butcher_results = list(/obj/item/clothing/head/crown = 1,)
+	response_help_continuous = "glares at"
+	response_help_simple = "glare at"
+	response_disarm_continuous = "skoffs at"
+	response_disarm_simple = "skoff at"
+	response_harm_continuous = "slashes"
+	response_harm_simple = "slash"
+	melee_damage_lower = 13
+	melee_damage_upper = 15
```

Comment:
don't forget to give these dudes object damage

maybe let ghosts possess them like spiders?

Reply:
ooh, and we could let ghosts possess the little rats as well, so they can suicide chew into strategic wires across the station

Reply:
no obj damage, they can ventcrawl

Reply:
yeah, I can understand not giving the little guys the ability to ventcrawl, but this guy is beeg

make his object damage below that required to damage an APC if you're really scared about griefing, but at least make this guy not able to be stymied by a table